net-vpn/eduvpn-client/eduvpn-client-4.1.3-r1.ebuild

```diff
@@ -35,4 +35,19 @@ RDEPEND="
 	>=net-vpn/eduvpn-common-1.1.2[${PYTHON_USEDEP}]
 "
 
-distutils_enable_sphinx doc
+PATCHES=(
+	"${FILESDIR}/${PN}-desktop.patch"
+)
+
+distutils_enable_sphinx doc \
+	dev-python/sphinx-rtd-theme
+
+pkg_postinst()
```

Comment:
Let's use the usual style of `x() {` please.

Reply:
fixed